### PR TITLE
Feat/php7.4

### PR DESCRIPTION
**Pull request by eiriksm**

### CODE REVIEW

**Comment by eiriksm** [comment]
Woah, quicker than the registry it seems ;)

**Review by andypost — APPROVED** [review]

File: 7.4/Dockerfile-cli [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM php:7.4
```
Comment (zaporylie) [inline 7.4/Dockerfile-cli]: Please use `php:7.4-alpine`

File: 7.4/Dockerfile-fpm [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM php:7.4
```
Comment (zaporylie) [inline 7.4/Dockerfile-fpm]: Please use `php:7.4-fpm-alpine`

**Review by zaporylie — CHANGES_REQUESTED** [review]

File: 7.4/Dockerfile-cli (hunk 1 above)
Comment (eiriksm) [reply 7.4/Dockerfile-cli]: Right, tested it on rc, and removed too much of the string for stable :)

**Review by zaporylie — APPROVED** [review]

**Pull request merged**